src/harness/fake_db_server.py

```diff
@@ -0,0 +1,157 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
Please either use this in the WDB test cases or remove it from this PR. I'd suggest the latter for expediency.

Reply:
this is an old file asked to be removed. We removed this file for now.